save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -124,7 +124,7 @@ class ProjectService(
      * @param userName
      * @return optional of user
      */
-    fun findUserByName(userName: String): Optional<User> = userRepository.findByName(userName)
+    fun findUserByName(userName: String): User? = userRepository.findByName(userName)
```

Comment:
Why saying no to `Optional`?

Reply:
`?` is more kotlin way for working with nullable variable

Reply:
Always thought `Optional`s is something from async world, but if there is no difference, will use `?` More often, thanks

Reply:
I changed only touch method in current PR